CHANGELOG.MD

```diff
@@ -0,0 +1,13 @@
+Summary of Code Changes for jQuery UI Multiselect Widget, Version 3.0.0:
```

Comment:
Don't need a changelog file, we'll detail all the enhancements when we release the version 3 tag.

Reply:
Ok.